frontend/packages/kubevirt-plugin/src/components/vms/virtualization.tsx

```diff
@@ -0,0 +1,64 @@
+import * as React from 'react';
+import { Helmet } from 'react-helmet';
+import { withStartGuide } from '@console/internal/components/start-guide';
+import { HorizontalNav } from '@console/internal/components/utils';
+import { VirtualMachinesPage } from './vm';
+import { VirtualMachineTemplatesPage } from '../vm-templates/vm-template';
+import { VirtualMachineModel } from '../../models';
+import { FLAGS } from '@console/shared';
+import { connectToFlags, FlagsObject } from '@console/internal/reducers/features';
+import { compose } from 'redux';
+
+export const WrappedVirtualizationPage: React.FC<VirtualizationPageProps> = (props) => {
+  const title = 'Virtualization';
+
+  const obj = { loaded: true, data: { kind: VirtualMachineModel.kind } };
+  const pages = [
+    {
+      href: '',
+      name: 'Virtual Machines',
+      component: VirtualMachinesPage,
+    },
+  ];
+
+  if (props.flags[FLAGS.OPENSHIFT]) {
+    pages.push({
+      href: 'templates',
+      name: 'Virtual Machine Templates',
+      component: VirtualMachineTemplatesPage,
+    });
+  }
+
+  return (
+    <>
+      <Helmet>
+        <title>{title}</title>
+      </Helmet>
+      <div className="co-m-nav-title">
+        <h1 className="co-m-pane__heading" data-test-id="cluster-settings-page-heading">
+          {title}
+        </h1>
+      </div>
+      <HorizontalNav
+        pages={pages}
+        match={props.match}
+        obj={obj}
+        customData={{ showTitle: false }}
+        {...props}
```

Comment:
nitpick: I think the convention is to put the props spread at the top of the props list.

Reply:
hmm, done !